src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -181,9 +181,9 @@ describe('Alerts (Unit)', () => {
           switch (url) {
             case `${safeConfigUrl}/api/v1/chains/${chain.chainId}`:
               return Promise.resolve({ data: chain });
-            case `${chain.transactionService}/api/v1/safes/${getAddress(
```

Comment:
As we are now fetching the Safe address via the Transaction Service and _not_ decoding it with `viem`, we do not need to checksum this here.